spiffworkflow-backend/pyproject.toml

```diff
@@ -33,6 +33,8 @@ sentry-sdk = "^1.10"
 # sphinx-autoapi = "^2.0"
 psycopg2 = "^2.9.3"
 typing-extensions = "^4.4.0"
+pymysql = "1.1.0"
```

Comment:
Same regard - is this a mac issue? if its commented out from __init__.py can we remove it from here also?